dz1-4.js

```diff
@@ -0,0 +1,34 @@
+function getRequest() {
+		if ( window.XMLHttpRequest) {
+			return new XMLHttpRequest();
+		} else if ( window.ActiveXObject) {
+			return new ActiveXObject("Microsoft.XMLHTTP");
+		}
+}
+
+function makeGETRequest (url) {
+		return new Promise ((resolve, reject) => {
+				
+		const xhr = this.getRequest();
```

Comment:
```suggestion
		const xhr = getRequest();
```
зачем обращаться к глобальному объекту?:)